bin/bitbake-setup

```diff
@@ -640,6 +638,10 @@ def create_settings(top_dir, non_interactive=True):
     with open(settings_path, 'w') as settingsfile:
         settings.write(settingsfile)
 
+def create_siteconf(top_dir, non_interactive=True):
+    siteconfpath = os.path.join(top_dir, 'site.conf')
+    print('A common site.conf file has been created, please edit or replace before running builds\n    {}\n'.format(siteconfpath))
```

Comment:
It's probably better to retain the notice that site.conf will be made, before anything is wrtten to disk?